src/java.desktop/share/classes/javax/swing/AbstractAction.java

```diff
@@ -81,14 +78,11 @@ public abstract class AbstractAction implements Action, Cloneable, Serializable
      * Whether or not to reconfigure all action properties from the
      * specified event.
      */
-    @SuppressWarnings("removal")
     static boolean shouldReconfigure(PropertyChangeEvent e) {
         if (e.getPropertyName() == null) {
             synchronized(AbstractAction.class) {
                 if (RECONFIGURE_ON_NULL == null) {
-                    RECONFIGURE_ON_NULL = Boolean.valueOf(
-                        AccessController.doPrivileged(new GetPropertyAction(
-                        "swing.actions.reconfigureOnNull", "false")));
+                    RECONFIGURE_ON_NULL = Boolean.getBoolean("swing.actions.reconfigureOnNull");
```

Comment:
This seems fine because the default value you were passing (false) is also the default of `Boolean.getBoolean`